src/bunnies/abstract.py

```diff
@@ -4,31 +4,32 @@
 
 
 class Bunny(ABC):
-    '''
+    """
     Abstract class for the bunnies
-    '''
+    """
 
     @abstractclassmethod
     def binarize(self, parameter):
-        '''
+        """
         binarize input parameters
-        '''
+        """
         pass
 
+    def swap_layers(self, model, verbose=False, *args, **kwargs) -> nn.Module:
 
-    def swap_layers(self, model, *args, **kwargs):
-        
         list_model = list(model.children())
         for idx, layer in enumerate(list_model):
             try:
                 layer.weight.data = self.binarize(layer.weight.data)
             except:
-                print(f"Cannot binarize")
+                if verbose:
```

Comment:
There's no place where `verbose` is taken as an argument.  
A better suggestion: `if kwargs.get(verbose, False):`  
What do you think?

Reply:
Done

Reply:
Cool